src/main/java/org/broadinstitute/hellbender/utils/QualityUtils.java

```diff
@@ -249,7 +258,7 @@ public static byte trueProbToQual(final double prob) {
      */
     public static byte trueProbToQual(final double trueProb, final byte maxQual) {
         if ( ! MathUtils.goodProbability(trueProb) ) throw new IllegalArgumentException("trueProb must be good probability but got " + trueProb);
-        final double lp = Math.round(-10.0*MathUtils.log10OneMinusX(trueProb));
+        final double lp = Math.round(LOG_PROB_TO_PHRED_MULTIPLIER * MathUtils.logOneMinusX(trueProb));
```

Comment:
i think the operation of `LOG_PROB_TO_PHRED_MULTIPLIER * something` needs to be extracted to a new method and used only that way throughout